plugins/ipam/dhcp/lease.go

```diff
@@ -432,7 +433,12 @@ func backoffRetry(resendMax time.Duration, f func() (*dhcp4.Packet, error)) (*dh
 
 		log.Print(err)
 
-		sleepTime = baseDelay + jitter(time.Second)
+		if fastRetryLimit == 0 {
+			sleepTime = baseDelay + jitter(time.Second)
+		} else {
+			sleepTime = jitter(time.Second)
```

Comment:
I think jitter(1s) is uniformly distributed in [-1, 1], so half the time this will end up being a sleep(0) which is probably not intended.

Reply:
Oh, thanks for pointing it out! Fixed in c9d0423.